src/TagHelperPack/MarkdownTagHelper.cs

```diff
@@ -45,8 +46,8 @@ public class MarkdownTagHelper : TagHelper
         /// (MIT license).
         /// </para>
         /// </remarks>
-        [HtmlAttributeName("normalize-indentation")]
-        public bool NormalizeIndentation { get; set; } = true;
+        [HtmlAttributeName("preserve-indentation")]
+        public bool PreserveIndentation { get; set; } = true;
```

Comment:
This should just default usual bool defaults and folks can set it if they want their indentation preserved.

```suggestion
        public bool PreserveIndentation { get; set; }
```